astropy/io/misc/asdf/tags/transform/__init__.py

```diff
@@ -1,2 +1,13 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 # -*- coding: utf-8 -*-
+
+import astropy.units as u
+
+__all__ = ['_parameter_to_value']
```

Comment:
This function shouldn't be used outside the module, so this really shouldn't be necessary, right?